packages/design-system/src/components/tooltip/index.js

```diff
@@ -106,6 +112,8 @@ function Tooltip({
   const [arrowDelta, setArrowDelta] = useState(null);
   const anchorRef = useRef(null);
   const tooltipRef = useRef(null);
+  const placementRef = useRef(placement);
+  const [realPlacement, setRealPlacement] = useState(placement);
```

Comment:
Just noodlin' - this naming makes sense to you? I was throwing a bunch of stuff at the wall to see where changes occurred, thus the origin of 'realPlacement'. When you read through it, does this make sense or should it be something different?

Reply:
hahaha, I saw it and giggled to myself. I will update now. I think I liked `dynamicPlacement`